data/curated/enmoma/english_monarchs_marriages_df.md

```diff
@@ -0,0 +1,7 @@
+|variable         |class     |description      |
+|:----------------|:---------|:----------------|
+|king_name        |character |king_name        |
```

Comment:
Edit the third column of each of these rows to add a description of the field.

Reply:
Yay! Thank you so much for your help @frankiethull.

Reply:
@lgibson7 thanks for the opportunity to contribute! I think this one is completed, glad to be the first person to try it out! All the steps are documented well I think I just have issues following directions :)

Reply:
I'm at posit conf, but I'll try to take a close look tonight.

I think our instructions might be a bit too wordy right now. We need to make it harder to misread 😊